test/recipes/90-test_store.t

```diff
@@ -223,24 +234,28 @@ sub init {
             # rsa-key-pkcs1.pem
             run(app(["openssl", "genrsa",
                      "-out", "rsa-key-pkcs1.pem", "2432"]))
-            # dsa-key-pkcs1.pem
-            && run(app(["openssl", "dsaparam", "-genkey",
-                        "-out", "dsa-key-pkcs1.pem", "1024"]))
-            # ec-key-pkcs1.pem (one might think that 'genec' would be practical)
-            && run(app(["openssl", "ecparam", "-genkey", "-name", "prime256v1",
-                        "-out", "ec-key-pkcs1.pem"]))
             # rsa-key-pkcs1-aes128.pem
             && run(app(["openssl", "rsa", "-passout", "pass:password", "-aes128",
                         "-in", "rsa-key-pkcs1.pem",
                         "-out", "rsa-key-pkcs1-aes128.pem"]))
+            && runall(sub {
+            # dsa-key-pkcs1.pem
+               run(app(["openssl", "dsaparam", "-genkey",
+                        "-out", "dsa-key-pkcs1.pem", "1024"]))
             # dsa-key-pkcs1-aes128.pem
             && run(app(["openssl", "dsa", "-passout", "pass:password", "-aes128",
                         "-in", "dsa-key-pkcs1.pem",
                         "-out", "dsa-key-pkcs1-aes128.pem"]))
+                      }, grep(/dsa-key-pkcs1\.pem$/, @generated_files))
```

Comment:
The indentation in this "sub" is quite confusing

Reply:
I'm trying to figure out why we need to use `runall` here (and similar occurrences) below. AFAICT we only ever run this sub once?.....ahh.......`dsa-key-pkcs1.pem`, may not exist in `@generated_files` at all - right? That's quite a convoluted way of writing an `||` condition, i.e. `!$use_dsa || run(app(....))`

Reply:
Good point; I've changed the guard accordingly.